index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+  <header>
+    <title> Bus Mall </title>
+  </header>
+
+  <body>
+    <div id="header"></div>
+    <img src="" id="Logo" />
+    <h1> Bus Mall Trials </h1>
+    <h2> The Products page has results </h2>
+    <nav>
+        <a href="index.html"> Home </a> 
+        <a href="products.html"> Products </a> 
+        <a href="about.html"> About  </a>         
+        <a href="contact.html"> Contact </a>
+  
+  <div class="tab">
+    <button class="tablinks" onclick="openPage(event, 'Home')"> Home </button>
+    <button class="tablinks" onclick="openPage(event, 'About')"> About </button>
+    <button class="tablinks" onclick="openPage(event, 'Contact')"> Contact </button>
+  </div>
+
+  <div id="Home" class="tabcontent">
+    <h3> Home </h3>
+    <p> This is text about the Home page </p>
+  </div>
+
```

Comment:
These should actually be pages at this point, not content blocks.

For this project, get a single section in the body to contain your <img> that will be randomly thrown in by your JS